config/7.5.1/common.yaml

```diff
@@ -7,8 +7,6 @@ console:
           application: "[[.ApplicationName]]"
           service: "[[.ApplicationName]]-[[.Console.Name]]"
       spec:
-        strategy:
-          type: Recreate
```

Comment:
@spolti instead of removing Recreate from common and adding everywhere else... maybe leave common as-is and just change the strategy in the couple specific envs that need to be rolling.

Reply:
I tried that, I got it as recreate strategy when I ran the tests.

Reply:
@tchughesiv What would you suggest give what @spolti experienced above? ^^

Reply:
@spolti @errantepiphany for example...
https://github.com/spolti/kie-cloud-operator/pull/1